src/ai/backend/manager/models/vfolder.py

```diff
@@ -1647,6 +1649,23 @@ async def resolve_num_files(self, info: graphene.ResolveInfo) -> int:
         "cur_size": ("vfolders_cur_size", None),
     }
 
+    async def resolve_service_config(self, info: graphene.ResolveInfo) -> str:
+        from ai.backend.manager.services.vfolder.actions.file import FetchServiceConfigAction
+
+        graph_ctx: GraphQueryContext = info.context
+
+        action_result = (
+            await graph_ctx.processors.vfolder_file.fetch_service_config.wait_for_complete(
+                FetchServiceConfigAction(
+                    vfolder_uuid=self.id,
+                    quota_scope_id=self.quota_scope_id,
+                    host=self.host,
+                    unmanaged_path=self.unmanaged_path,
+                )
+            )
+        )
+        return json.dumps(action_result.result.model_dump())
```

Comment:
Does this look like a weird data structure? It looks like you're holding the DTO directly in the action result right now, which doesn't seem appropriate.